RULES.md

```diff
@@ -0,0 +1,150 @@
+
+Make [based.cooking](https://based.cooking) based again!
+
+## Rules
+
+### Freedom
+
+#### License
+
+When you submit anything to [based.cooking](https://based.cooking), you agree to release your work under the CC0 1.0 license.
+Everything in the [based.cooking](https://based.cooking) repository is licensed under the CC0 1.0 license.
+Contributions licensed under any incompatible license will be rejected.
+
+#### Requests for removal
+
+Requests to have your work removed will be ignored.
+
+#### Attribution
+
+Attribution is unnecessary under the CC0 1.0 license.
+However, you are welcome and recommended to give yourself credit at the bottom of a recipe you submitted, including links to your personal website and to your donation pages.
+**Do not** link to social media sites.
+
+### Recipe format
+
+Model submission files after [example.md](example.md), then put them in `src/`.
+
+### Markdown style
+
+Recipes should start with a title, with a single `#` *on the first line*.
+
+### Technical details
+
+#### File format
+
+- Recipes should start with no empty line at the top and end with no trailing lines.
+- There should be no trailing whitespace.
+- The file needs to be `\n` terminated in Unix-like fashion (if you're using a Unix-like system, it should be automatic).
+- Encode files in UTF-8.
+
+#### File name
+
+- File names should be the name of the dish with words separated by hyphens (`-`), not underscores, and definitely not spaces.
+
+#### Tags
+
+- Recipes should end with a list of relevant tags.
+The syntax is:
+
+```md
+;tags: tag1 tag2 tag3
+```
+
+- The tag line should be a single line, at the end of the markdown file, preceded by a blank line.
+- Add between a few (1-4) tags.
+- **Prioritize existing tags**.
+- As a general guideline, add the country from which the recipe originates if the recipe is representative of said country, using the adjective form (eg. *mexican*, *italian*, etc).
+- Tag the main ingredient if it's something slightly special.
+
+##### List of special, categorical tags to use if relevant
+
+- `basic`: for basic recipes that aren't meant to be standalone but are supposed
+  to be incorporated in another recipe.
+- `breakfast`
+- `cheesefare`: for recipes that include dairy but no milk.
+- `dessert`
+- `drink`
+- `quick`: for recipes that can be cooked in under ~20 minutes.
+- `side`: side dishes such as mash, fries, etc.
+- `snack`
+- `spread`
+- `strict-fast`: for recipes that contain no meat or dairy.
+- `vegan`
+- `vegetarian`
+
+#### Images
+
+- Images are stored in `data/pix`.
+- Each recipe can have a title image at the top and perhaps several instructional images as absolutely necessary.
+- Do not add stock images you found on the internet.
+Take a good picture yourself of the actual dish you prepared.
+- If you see a bad or substandard image, you may submit a better one.
+- Images should be in `.webp` format and with as small a file size as possible.
```

Comment:
CI script currently rejects images over 150kB, so we could mention that here as well... Or make that lower if 150 is deemed to high

Reply:
I added a rule with "Kb" but just to be sure, are we talking about kilobytes or kibibytes?

Relevant [xkcd](https://xkcd.com/394/).

Reply:
it is 150000 in the code, so it is kilobytes. I have not seen that xkcd before :D

Reply:
In that case your "kB" was correct.

Reply:
Fixed in recent commit.